hi-ml-cpath/src/health_cpath/configs/classification/BaseMIL.py

```diff
@@ -32,6 +33,14 @@
 from health_cpath.utils.tiles_selection_utils import TilesSelector
 
 
+# Pytorch Lightning prints a warning if the batch size is not consistent across all batches. The way PL infers
+# the batch size is not compatible with our data loaders. It searches for the first item in the batch that is a
+# tensor and uses its size[0] as the batch size. However, in our case, the batch is a list of tensors, so it
+# thinks that the batch size is the bag_size which can be different for each WSI in the batch. This is why we
+# ignore this warning to avoid noisy logs.
+warnings.filterwarnings("ignore", ".*Trying to infer the `batch_size` from an ambiguous collection.*")
+
+
```

Comment:
Can we put that into the constructor of BaseMIL? If here, it will potentially affect more models - because it triggers any time somebody imports BaseMIL.